tests/behave/features/environment.py

```diff
@@ -1,10 +1,11 @@
+import os
 from selenium import webdriver
 from selenium.webdriver.firefox.options import Options
 
 
 def before_feature(context, feature):
     options = Options()
-    options.headless = True
+    options.headless = os.environ.get("SELENIUM_HEADLESS", "True") == "True"
```

Comment:
Consider converting this to `True` (boolean) or an existence check instead of checking equality to `"True"` -- otherwise users have to remember to put `SELENIUM_HEADLESS=True` in this exact format and case.